crates/cast/bin/cmd/run.rs

```diff
@@ -48,6 +48,14 @@ pub struct RunArgs {
     #[arg(long)]
     quick: bool,
 
+    /// Prints the trace in JSON format.
+    #[arg(long, short)]
+    json: bool,
```

Comment:
We've recently added the global `shell::is_json()` to access the global `--json` flag